### ISSUE TITLE
Publish new version of chrome extension

### ISSUE THREAD

**Issue by paulirish** [GoogleChrome/lighthouse#127]
The one in CWS is still based on the gist. Been a while, but I think the state of the master is good enough for CWS.

**Comment by brendankenny** [comment]
yeah, with #126 added, this would be great to do.

**Comment by paulirish** [comment]
Published `0.0.6` from master just now.

Last commit: 

```
commit 843d145e0b13099bb3b34078a61930e6a37d01f9
Author: Paul Lewis <gravatar@aerotwist.com>
Date:   Mon Apr 4 22:46:12 2016 +0100

    Adds banner to header.
```


![image](https://cloud.githubusercontent.com/assets/39191/14265721/0311ab4a-fa78-11e5-86fc-5490caef338e.png)
 
Our 72(!) users will get a new build within hours. \o/
